lib/skylight/railtie.rb

```diff
@@ -117,7 +117,11 @@ def environments
     end
 
     def activate?
-      environments.include?(Rails.env.to_s)
+      environments.include?(Rails.env.to_s) || env_enabled?
+    end
+
+    def env_enabled?
+      ENV.key?("SKYLIGHT_ENABLED") && ENV["SKYLIGHT_ENABLED"] !~ /^false|no$/i
```

Comment:
I think that if it is explicitly set to `false` then we might want that to override the `environments` list. I'd fall back to the environments list only if the ENV var isn't set. Also, I was looking at our other code for environment variables and we don't handle "no" anywhere. So it's ok to change the regex to just look for "false".

Reply:
Fixed and added tests for explicit disable case